discovery-infra/tests/conftest.py

```diff
@@ -90,6 +90,13 @@ def get_kubeconfig_path():
     env_variables["vip_dhcp_allocation"] = False
 
 
+if env_variables["is_ipv6"]:
```

Comment:
Should be part of the triggers dict after https://github.com/openshift/assisted-test-infra/pull/860 would be merged